### PR TITLE
Adding link to more info

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1524

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/disable-basic-authentication-in-exchange-online.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ title: Disable Basic authentication in Exchange Online
 
 > [!NOTE]
 > If you've enabled _security defaults_ in your organization, Basic authentication is already disabled in Exchange Online. For more information, see [What are security defaults?](https://docs.microsoft.com/azure/active-directory/conditional-access/concept-conditional-access-security-defaults).
+> Please see [Basic Authentication and Exchange Online](https://techcommunity.microsoft.com/t5/exchange-team-blog/basic-authentication-and-exchange-online-july-update/ba-p/1530163) for the latest announcements concerning Basic Authentication.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/disable-basic-authentication-in-exchange-online.md:26]:
```suggestion
> Please see [Basic Authentication and Exchange Online](https://techcommunity.microsoft.com/t5/exchange-team-blog/basic-authentication-and-exchange-online-july-update/ba-p/1530163) for the latest announcements concerning Basic authentication.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@get-itips - Your capitalization is correct, but "Basic Authentication" has been consistently capitalized incorrectly throughout the topic, so for the sake of consistency I would suggest this edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**